### DISCUSSION TITLE
Automatically Refresh Data View After SQLite Updates

### DISCUSSION THREAD

**Discussion by sagarspatil** [damms005/devdb-vscode#41] [Ideas]
Whenever I execute my Python code to modify data in my SQLite database, will the 'devdb' automatically refresh and display the updated data in the data view?

**Comment by damms005** [comment]
Currently, no. You'll have to click on the refresh icon

![image](https://github.com/damms005/devdb-vscode/assets/9839355/3f7b7e25-01db-4b5c-8a23-a2b4fe6f21ea)

But I agree with you this is a good feature to add, at least for SQLite. Basically the database file can be watched for changes and whenever there is a change data should be reloaded to the UI

You can open a PR if you can or I'll give it a shot sometimes later.
